runtime/doc/faq.txt

```diff
@@ -0,0 +1,684 @@
+*faq.txt*		 Frequently Asked Questions
+
+                            NVIM REFERENCE MANUAL
+
+                                  Type |gO| to see the table of contents.
+
+==============================================================================
+1. General Questions                                 *faq* *faq-general-questions*
```

Comment:
I think we should not number things manually. Let `gO` do that, or just live without it.
```suggestion
General Questions                                 *faq* *faq-general-questions*
```